docusaurus/static/recipes-assets/splash-screen/SplashScreen.module.css

```diff
@@ -0,0 +1,41 @@
+@import "../../styles/imports/variables";
+
+:root {
+    --z-index-max: 2147483647;
```

Comment:
Is just the MAX_INT allowed by browsers. The splash screen should be above everything, that's why I put this.

Reply:
I understand, but I think this needs to be a more conservative number to prevent developers from using random large numbers when they should be using sequential numbers and this should be the highest. I suggest add a comment for example.

`-z-index-max: 5;  /* This should be the highest z-index value in the project */`

Reply:
Also, giving the base container of the `SplashScreen` the highest possible `z-index` leaves no space for using different z-indexes in elements inside the `SplashScreen` Component.

Since these would be simple components, perhaps this would be a rare situation, but i think it should be taken into account.